straxen/rundb.py

```diff
@@ -206,16 +200,16 @@ def _find(self, key: strax.DataKey,
                         'host': self.hostname,
                         'type': key.data_type,
                         'protocol': strax.FileSytemBackend.__name__,
```

Comment:
Is this field used anywhere else? Since we are updating things maybe we could consider removing this as well?

Reply:
Yes, we use it later to say that it's the `FileSytemBackend` here:
https://github.com/XENONnT/straxen/blob/5d20f66aab548657e39bad57cb620a2ea7c9594a/straxen/rundb.py#L255
and here
https://github.com/XENONnT/straxen/blob/5d20f66aab548657e39bad57cb620a2ea7c9594a/straxen/rundb.py#L255

We could hardcode `'FileSytemBackend'` if needed but I would argue to leave it as it is